### DISCUSSION TITLE
System.Speech 5.0.0 with Linux?

### DISCUSSION THREAD

**Discussion by thenicnic** [dotnet/runtime#53824] [Q&A] [answered]
Does System.Speech 5.0.0 work with Linux, too? 
[.NET Core version has been released a few weeks ago](https://www.nuget.org/packages/System.Speech), but VS 2019 tells me that the only supported platform is "windows".

**Comment by FiniteReality** [comment, marked as answer]
No. System.Speech relies on the Windows text-to-speech and speech-to-text APIs and it won't work on other platforms.

**Reply by thenicnic** [reply]
Alright, thanks for letting me know Monica! But will it run on Windows 10 IoT for Small Devices on Raspberry Pi 4?

**Reply by danmoseley** [reply]
@thenicnic it depends whether that SKU has the SAPI API's. Given they're somewhat legacy, and presumably they're optimizing for size, I'm guessing they aren't there. Quickest way to find out is to do this, if you have Powershell:

https://github.com/dotnet/runtime/issues/46730#issuecomment-782928477

**Reply by danmoseley** [reply]
If that doesn't work, let me know and I'll reach out to Speech folks to ask what options they are. If your device has connectivity, the Azure Speech service is one.

**Comment by danmoseley** [comment]
If someone knows of a .NET speech library (at least synthesis?) that works on Linux, this would  be a good place to post info.
